db/migrate/20170203113625_remove_organisation_reference_from_content_items.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveOrganisationReferenceFromContentItems < ActiveRecord::Migration[5.0]
+  def change
+    remove_reference :content_items, :organisation, foreign_key: true
```

Comment:
We discussed this earlier on Friday and I think Hong didn't add this to the checklist..

Since the data is can be derived at this stage.. it appeared low risk to simple drop and regenerated data.